test/fuzz_group.cpp

```diff
@@ -286,6 +290,7 @@ void parse_and_apply_instructions(std::string& in, const std::string& path, util
 
         for (;;) {
             char instr = get_next(s) % COUNT;
+            iteration++;
```

Comment:
Is this just so you can inspect it in a debugger? It doesn't seem to have any use.

Reply:
Ues. Added a comment.